h2o-core/src/main/java/water/util/Log.java

```diff
@@ -339,7 +341,16 @@ private static void setLog4jProperties(String logDir, java.util.Properties p) th
     String h2oLog4jConfiguration = System.getProperty ("h2o.log4j.configuration");
 
     if (h2oLog4jConfiguration != null) {
-      PropertyConfigurator.configure(h2oLog4jConfiguration);
+      // Try to configure via a file on local filesystem
+      if (new File(h2oLog4jConfiguration).exists()) {
+        PropertyConfigurator.configure(h2oLog4jConfiguration);
+      } else {
+        // Try to load file via classloader resource (e.g., from classpath)
+        URL confUrl = Log.class.getClassLoader().getResource(h2oLog4jConfiguration);
+        if (confUrl != null) {
+          PropertyConfigurator.configure(confUrl);
+        }
```

Comment:
It would be good to print an error if the user specified a config but we didn't use it because it was invalid. Wdyt?